omnibus/config/software/datadog-agent.rb

```diff
@@ -16,6 +16,9 @@
 
 dependency "openscap" if linux_target? and !arm7l_target? and !heroku_target? # Security-agent dependency, not needed for Heroku
 
+# Alternative memory allocator
+dependency "libjemalloc" if linux_target?
```

Comment:
@StephenWakely I added this (addresses [this comment](https://github.com/DataDog/datadog-agent/pull/29628/files#r1778970772)); feel free to add more context to the comment and also let me know if my assumption that this is linux only is not correct (or whether there's any other platform-specific condition that we may need).

Reply:
Linux only is correct. We need more testing on Windows before deciding if it needs to be installed there too. My hunch is that we won't need to.